orodha-base-image/dev-requirements.txt

```diff
@@ -31,4 +31,3 @@ tomli==2.0.1
 Werkzeug==2.3.6
 zipp==3.15.0
 pytest-mock==3.11.1
-orodha-keycloak==0.5
```

Comment:
@Edison-Stuart Will we not need the package for all envs?

Reply:
I'll resolve this for now, but I think you may find yourself revisiting this later.